### DISCUSSION TITLE
Why did the embedding format changes happen?

### DISCUSSION THREAD

**Discussion by MhMdRiyazath** [spring-projects/spring-ai#1221] [Q&A] [answered]
Previously, the getOutput method returned a list of doubles. However, it has now been changed to return a list of floats, even though OpenAI models still return a list of doubles

**Comment by sobychacko** [comment, marked as answer]
See this PR for more details: https://github.com/spring-projects/spring-ai/pull/1002
and the corresponding issue: https://github.com/spring-projects/spring-ai/issues/405
